hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestInsertTable.scala

```diff
@@ -303,5 +304,184 @@ class TestInsertTable extends TestHoodieSqlBase {
       "assertion failed: Required select columns count: 4, Current select columns(including static partition column)" +
         " count: 3，columns: (1,a1,10)"
     )
+    spark.sql("set hoodie.sql.bulk.insert.enable = true")
+    spark.sql("set hoodie.sql.insert.mode= strict")
+
+    val tableName2 = generateTableName
```

Comment:
Can we also enhance the test w/ both type of partitions(single level and multi-level).

Reply:
Yes, will add the case for multi-level partition